docker/apps/agreement_database/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM vickumar/python-nodejs
+
+RUN git clone https://github.cfpb.gov/CFPB/agreement_database.git /apps/agreement_database/
```

Comment:
@vickumar1981 would it be possible to instead have the individual projects have their own `Dockerfile`s? See for example how the HMDA grasshopper project has a main [docker-compose-common.yml](https://github.com/cfpb/grasshopper/blob/master/docker-compose-common.yml) that refers to sub-projects (that are supposed to be checked out within the main project directory). Their sub-projects have their own Dockerfiles, for example [grasshopper-parser](https://github.com/cfpb/grasshopper-parser/blob/master/Dockerfile).